### PR TITLE
Projects

### PR DESCRIPTION
**Pull request by AymanAlqoqa**
Projects table is created with its functionality except fetching data from database
relates #17

### CODE REVIEW

File: client/src/components/commonComponents/search/index.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
```
Comment (ashatat) [inline client/src/components/commonComponents/search/index.jsx]: do this in a separate PR, please.

File: client/src/App.js [hunk 2]
```diff
@@ -1,18 +1,34 @@
-import React, { Component } from "react";
-import { BrowserRouter, Route, Switch, Redirect } from "react-router-dom";
-import "./App.css";
+import React, { Component } from 'react';
```
Comment (ashatat) [inline client/src/App.js]: don't commit changes on files outside your component unless it's related to your PR directly

File: client/src/components/Projects/projectsTable/index.jsx [hunk 3]
```diff
@@ -0,0 +1,156 @@
+import React, { Component } from 'react';
+import { Link, Redirect } from 'react-router-dom';
```
[the remark this answers is not in the document]
Comment (AymanAlqoqa) [reply client/src/components/Projects/projectsTable/index.jsx]: I removed it, 
thanks

**Comment by AymanAlqoqa** [comment]
> Good work
> 
> I have more few comments:
> 1- when you click on edit the rowSelected will be null if you didn't click directly on the ... icon
> so I'd suggest you change it on hover?
> 2- Also clicking on the project (the row) should take me to `project/:projectId`

1- rowSelected is done.
2- I think it is better to click on project name not on the row !!!
thanks

File: client/src/App.js (hunk 2 above)
Comment (AymanAlqoqa) [reply client/src/App.js]: ok,
thanks

File: client/src/components/commonComponents/search/index.jsx (hunk 1 above)
Comment (AymanAlqoqa) [reply client/src/components/commonComponents/search/index.jsx]: ok thanks
